lib/code_corps/policy/policy.ex

```diff
@@ -92,7 +95,7 @@ defmodule CodeCorps.Policy do
     def can?(%User{} = user, :create, Skill), do: Policy.Skill.create?(user)
 
     def can?(%User{} = user, :show, %StripeConnectAccount{} = stripe_connect_account), do: Policy.StripeConnectAccount.show?(user, stripe_connect_account)
-    def can?(%User{} = user, :create, %Changeset{ data: %StripeConnectAccount{}} = changeset), do: Policy.StripeConnectAccount.create?(user, changeset)
+    def can?(%User{} = user, :create, %StripeConnectAccount{}, params), do: Policy.StripeConnectAccount.create?(user, params)
     def can?(%User{} = user, :update, %StripeConnectAccount{} = stripe_connect_account), do: Policy.StripeConnectAccount.update?(user, stripe_connect_account)
```

Comment:
The end result is that you want to remove lines 97, 98 and 99 in favor of lines 54, 55 and 56